appveyor.yml

```diff
@@ -23,9 +23,12 @@ init:
 # cloning the repository happens here
 
 install:
-  - set PATH=C:\Ruby26-x64\bin;%PATH%
-  - FOR /F "usebackq delims==" %%A IN (`ruby .ci/compatible_gem_version`) DO gem update --system %%A
-  - FOR /F "usebackq delims==" %%A IN (`ruby .ci/bundler_version.rb`) DO gem install bundler -v %%A
+  - set PATH=C:\Ruby27-x64\bin;%PATH%
+  - ruby --version
+  - ruby .ci/compatible_gem_version
+  - FOR /F "usebackq delims==" %%A IN (`ruby .ci/compatible_gem_version`) DO gem update --no-document --system %%A
+  - ruby .ci/bundler_version.rb
+  - FOR /F "usebackq delims==" %%A IN (`ruby .ci/bundler_version.rb`) DO gem install bundler --no-document -v %%A
```

Comment:
- Add version info before trying to update. Was helpful while it was failing to use Ruby 2.7 on the older images.
- Add `--no-document` to speed it up a little bit.